src/internal/Subscription.ts

```diff
@@ -85,7 +69,7 @@ export class Subscription implements SubscriptionLike {
           } catch (err) {
             errors = errors ?? [];
             if (err instanceof UnsubscriptionError) {
-              errors = [...errors, ...err.errors];
+              errors.push(...err.errors);
```

Comment:
Previously, we were unnecessarily creating a new array every time we ran into an `UnsubscriptionError` inside of our unsubscription. This is probably a corner case, but it's silly that we were doing that.